tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py{27,34,35,pypy}
+
+[testenv]
+usedevelop = True
+setenv =
+    PYTHONPATH = {toxinidir}
+basepython =
+    py27: python2.7
+    py34: python3.4
+    py35: python3.5
+    pypypy: pypy
+deps =
+    -r{toxinidir}/requirements.txt
+commands =
+    py.test --cov=src/barbequeue --color=no {posargs}
```

Comment:
I'm leaning on turning off `usedevelop`: http://tox.readthedocs.io/en/latest/config.html#confval-usedevelop=BOOL

In order to test the installed version of bbq, not the symlinked version.